products/ga_s2_gm.odc-product.yaml

```diff
@@ -1,4 +1,4 @@
-name: ga_s2_gm
+name: gm_s2_annual
 description: Annual Geometric Median, Sentinel-2
```

Comment:
Can we please update this too

Reply:
Done.

Reply:
This should be something like:

> Surface Reflectance Annual Geometric Median and Absolute Deviations, Sentinel-2

It's not just the geomedian anymore.